boltzr/src/api/mod.rs

```diff
@@ -130,6 +130,10 @@ where
                 get(get_stats::<S, M>),
             )
             .route("/v2/swap/rescue", post(swap_rescue::<S, M>))
+            .route(
+                "/v2/swap/rescue/extended",
```

Comment:
```suggestion
                "/v2/swap/restore",
```

How about calling it "restore"? Our goal here is effectively restoring the state of the client to where it was before all its data was deleted